docs/plans/ai-analytics-insights.md

```
@@ -0,0 +1,934 @@
+| `apps/web/lib/db/schema/enums.ts` | Add 5 new enums |
+| `apps/web/lib/db/schema/index.ts` | Export `./insights` |
+| `apps/web/constants/routes.ts` | Add `INSIGHTS: '/app/insights'` |
+| `apps/web/lib/queries/keys.ts` | Add `insights` query key factory |
+| Dashboard layout/sidebar | Add "Insights" nav item with badge |
+| Dashboard overview page | Add InsightsSummaryWidget |
+
+---
+
+## 11. Implementation Phases
+
+### Phase 1: Foundation (Schema + Aggregation)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)